src/common.h

```diff
@@ -144,3 +144,12 @@
     ((type *) ((char *) (ptr) - (offsetof(type, member))))
 #endif
 #endif
+
+static inline unsigned int align_up(unsigned int sz, unsigned long alignment)
```

Comment:
Be aware of derived types. Should be
```c
static inline uintptr_t align_up(uintptr_t sz, size_t alignment)
```

Reply:
If we want to use `uintptr_t` and `size_t`, we need to include other header file in common.h.

Reply:
> If we want to use `uintptr_t` and `size_t`, we need to include other header file in common.h.

Alternatively, implement `align_up()` in `src/utils.h`.